proto/data.proto

```diff
@@ -252,6 +252,29 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is a unix nanos timestamp and is set when requesting the
+  // lease according to the wall clock plus the Duration below at the lease
+  // requestor / grantee, which is also the only node that uses it directly.
+  // Granters must use always substitute their local walltime plus the
+  // Duration below instead.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
+  // The duration, specified in unix nanos, is the duration for which lease
+  // granters guarantee not to participate in elections, beginning right
+  // after the command has been committed and applied.
```

Comment:
I think this can be "beginning after the command has been accepted". The proposer of the lease marks time from the point that it first proposes to Raft. All acceptors can mark duration from their wall clocks at time of acceptance, as that interval is guaranteed to end in absolute time _after_ the expiration at the original proposer, as propose time is strictly before any acceptance time.